docs/examples/use_cases/webdataset-externalsource.ipynb

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
<p>I'd title it something like "WebDataset integration with External Source"</p><p> </p><p>Then update this first sentence:</p><p> </p><p>one may combine the webdataset with a DALI pipeline, using a external source operator</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>